internal/controllers/clusterextension_controller_test.go

```diff
@@ -196,70 +195,10 @@ func TestClusterExtensionResolutionSuccessfulUnpackFails(t *testing.T) {
 	}
 }
 
-func TestClusterExtensionUnpackUnexpectedState(t *testing.T) {
-	cl, reconciler := newClientAndReconciler(t)
-	reconciler.Unpacker = &MockUnpacker{
-		result: &source.Result{
-			State: "unexpected",
```

Comment:
Because the rukpak `source.Result` struct is gone and there is no concept of unpack state anymore, this entire test is no longer relevant.